include/avif/internal.h

```diff
@@ -15,6 +15,9 @@ extern "C" {
 #define AVIF_MIN(a, b) (((a) < (b)) ? (a) : (b))
 #define AVIF_MAX(a, b) (((a) > (b)) ? (a) : (b))
 
+// Returns the equivalent of ceil(a/b) but for positive integers.
+#define AVIF_CEIL(a, b) (((a) + (b)-1) / (b))
```

Comment:
Name this macro `AVIF_CEIL_DIV` or `AVIF_DIV_CEIL` because it also does a division.

Add spaces around the `-` operator: `(((a) + (b) - 1) / (b))`

Reply:
> Name this macro `AVIF_CEIL_DIV` or `AVIF_DIV_CEIL` because it also does a division.

Done.

> Add spaces around the `-` operator: `(((a) + (b) - 1) / (b))`

This is a bug of clang-format. See https://github.com/llvm/llvm-project/issues/41502.